fake-image

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Make a `scripts/` dir, put this in it, amend the file extension (`.sh`) and leave a comment explaining what it does in the file please

Reply:
Done